packages/react-components/lib/systems-summary/system-summary-type.tsx

```diff
@@ -0,0 +1,41 @@
+import { Notification } from '../index';
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
+
+export interface ItemSummaryState {
+  operational: number;
+  outOfOrder: number;
+  idle?: number;
+  charging?: number;
+}
+
+export interface ItemSummary {
+  item: string;
+  summary: ItemSummaryState;
+  outOfOrder: string[];
+}
+
+export interface MainMenuItemStateProps {
```

Comment:
I would prefer to put these interfaces in the same file as the components, @matiasbavera do you have any preference?